ravenframework/Databases/Database.py

```diff
@@ -121,22 +119,21 @@ def _handleInput(self, paramInput):
         self.exist = False
       self.initializeDatabase()
     else:
-      #file does not exist in path
+      # file does not exist in path
       if self.readMode == 'read':
         self.raiseAnError(IOError, f'Requested to read from database, but it does not exist at "{fullpath}"; '+
                           'The path to the database must be either absolute, or relative to <workingDir>!')
       self.exist = False
       self.initializeDatabase()
     self.raiseAMessage(f'Database is located at "{fullpath}"')
 
-  def initialize(*args, **kwargs):
+  def initialize(self, *args, **kwargs):
     """
       Initialization for data object, if any.
       @ In, args, list, ordered arguments
       @ In, kwargs, dict, keyword arguments
       @ Out, None
     """
-    pass
```

Comment:
I like the "pass" for explicit clarity, but to preference.

Reply:
VS Code with `pylint` set up following the Wiki instructions doesn't like it. I tried to clean up as many of the `pylint` "problems" as I could and this is one of them.